app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -114,10 +117,22 @@ class DefaultBrowserToolbarController(
                 currentSession?.id
             )
 
-            navController.nav(R.id.browserFragment, directions)
+            navController.nav(R.id.browserFragment, directions, getToolbarNavOptions())
         }
     }
 
+    private fun getToolbarNavOptions(): NavOptions {
```

Comment:
Is this something that could live as a static function in `BrowserAnimator` so we don't have to dup the code in these two places? :)

Reply:
Yes, thank you, i thought i should put it somewhere global but i didn't know where 😀, I'm working on it thank you for your time 😊